### ISSUE TITLE
Text background white and foreground black in .txt and unknown (.mp3, .exe, etc) files with NppCalc plugin

### ISSUE THREAD

**Issue by karlkoorna** [Nidre/VS2015-Dark-Npp#28]
It's only happening while NppClac is installed. I would like to run them both if possible.

Notepad++ 7.3.1 (32-bit)
NppClac 1.4

![image](https://cloud.githubusercontent.com/assets/24987433/22400725/cdf79110-e5c5-11e6-8f0e-ea54c1c4efb4.png)

**Comment by sibunanick** [comment]
Yeah look in to this people, this is bottering me for such a long time.

**Comment by sippinonwhat** [comment]
yepp!!
